lnst/RecipeCommon/Perf/Evaluators/BaselineEvaluator.py

```diff
@@ -53,8 +58,23 @@ def evaluate_group_results(
             )
             comparison_result = ResultType.max_severity(comparison_result, comparison)
             result_text.extend(text)
+            comparisons.append(
+                {
+                    "current_result": result,
+                    "baseline_result": baseline,
+                    "comparison_result": comparison,
+                    "text": text,
+                    "recipe_conf": recipe_conf,
```

Comment:
hmm, is it necessary to keep `recipe_conf` for every result? afaik many of results have common`recipe_conf`

Reply:
sure, but if it is the same object then object should only exist once, this will only add a reference to that object for convenience, it doesn't duplicate it

it's also non-trivial to find the perf recipe configuration related to a specific measurement result without this at this point and i'm not sure if we would want to add it into some form of a grouped result...

Reply:
is this still relevant or can we resolve this @enhaut ?

Reply:
hmm, im ok with that